lib/ansible/plugins/action/eric_eccli_command.py

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2019 Ericsson AB.
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.plugins.action.normal import ActionModule as _ActionModule
+
+
+class ActionModule(_ActionModule):
+
+    def run(self, tmp=None, task_vars=None):
+        if self._play_context.connection != 'network_cli':
+            return {'failed': True, 'msg': 'Connection type %s is not valid for cli_config module' % self._play_context.connection}
```

Comment:
This action plugin is not required since the module will support the only conenction=network_cli. You can add the connection type supported in the module documentation.
Refer https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/junos/junos_command.py#L95